lib/proxy/helpers.rb

```diff
@@ -25,7 +25,8 @@ def log_halt code=nil, exception=nil
 
   # read the HTTPS client certificate from the environment and extract its CN
   def https_cert_cn
-    certificate_raw = request.env['SSL_CLIENT_CERT'].to_s
+    ssl_client_cert_header = Proxy::SETTINGS.ssl_client_cert_header || 'SSL_CLIENT_CERT'
+    certificate_raw = request.env[ssl_client_cert_header].to_s.gsub(/   /, "\n")
```

Comment:
This is a known issue with nginx: https://forum.nginx.org/read.php?2,236546,236546#msg-236546.

As header continuation can use either a single space or a tab, therefore something like this would be a better way for stripping of whitespaces:

```request.env[Proxy::SETTINGS.ssl_client_cert_header].to_s.gsub(/\s+/, "\n")```

Lastly, could you add a comment explaining a reason for this line please.